autoshop/api/v1/views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.generics import ListAPIView, RetrieveAPIView, UpdateAPIView, DestroyAPIView
+
+from autoshop.api.v1.serializers import UserProfileSerializer
+from core.models import Profile
+
+
+class ProfileList(ListAPIView):
```

Comment:
Well its about what you want to do with end point. We want an endpoint for django model with the basic crud operations. Although we will not support the delete operation for now.